sample/build.gradle

```diff
@@ -2,8 +2,8 @@ apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
```

Comment:
Maybe we could just remove the whole sample? 🤔

Reply:
not a big deal to maintain it and it is sometimes useful to build it locally and check out the sample